apps/fronttoback/studio/schemas/documents/question.js

```diff
@@ -46,7 +51,14 @@ export default {
   ],
   preview: {
     select: {
-      title: 'questionId',
+      label: 'label',
+      questionId: 'questionId',
+    },
+    prepare(selection) {
+      const {label, questionId} = selection
+      return {
+        title: `[${questionId}] ${label}`,
```

Comment:
I had a very hard time correlating convertkit sequences, tags, etc with the sanity models. we need to add human labels to these and make it obvious what goes to what when possible